contrib/terraform/openstack/group_vars/all.yml

```diff
@@ -105,21 +105,18 @@ kube_apiserver_insecure_port: 8080 # (http)
 # into appropriate IP addresses. It's highly advisable to run such DNS server,
 # as it greatly simplifies configuration of your applications - you can use
 # service names instead of magic environment variables.
-# You still must manually configure all your containers to use this DNS server,
-# Kubernetes won't do this for you (yet).
 
-# Do not install additional dnsmasq
-skip_dnsmasq: false
```

Comment:
what's why I proposed to not change `skip_dnsmasq` logic, it has nothing to the hosts/pods config.
Please keep this var and use `dns_mode` to only control if hosts' DNS config will be managed or docker one, and also to supersede `dns_setup`